diff --git a/src/libp2p/Libp2p.Core/Discovery/PeerStore.cs b/src/libp2p/Libp2p.Core/Discovery/PeerStore.cs
index a6ea66b..3b2abaf 100644
--- a/src/libp2p/Libp2p.Core/Discovery/PeerStore.cs
+++ b/src/libp2p/Libp2p.Core/Discovery/PeerStore.cs
@@ -21,7 +21,7 @@ public class PeerStore
 
         if (!SigningHelper.VerifyPeerRecord(signedEnvelope, publicKey, out _))
         {
-            return;
+            // return; // [PATCH] Accept unverified Python PeerRecords during interop testing
         }
 
         Multiaddress[] addresses = PeerRecord.Parser.ParseFrom(signedEnvelope.Payload).Addresses
